app/models/report.rb

```diff
@@ -0,0 +1,18 @@
+class Report < ActiveRecord::Base
+  belongs_to :partner
+  belongs_to :uploaded_by, class_name: "Publisher"
```

Comment:
I believe it's `Partner` rather than Publisher.

Reply:
Ah, so both partners and admins will upload reports so I made this publisher since it refers to both